src/main/groovy/netflix/nebula/dependency/recommender/provider/RecommendationProviderContainer.java

```diff
@@ -174,4 +177,12 @@ public Boolean isStrictMode() {
     public void setStrictMode(Boolean strict) {
         strictMode = strict;
     }
+    
+    public void excludeConfigurations(String ... names) {
+        excludedConfigurations.addAll(Arrays.asList(names));
+    }
+
+    public Set<String> getExcludedConfigurations() {
```

Comment:
What about `setExcludedConfigurations`, to support things like `excludedConfigurations += otherList`?